docs/man/rpm-philosophy.7.scd

```diff
@@ -0,0 +1,163 @@
+RPM-PHILOSOPHY(7)
+
+# NAME
+rpm-philosophy - RPM's design philosophy
+
+# DESCRIPTION
+RPM is a general purpose software manager. It has
+two major parts: rpmbuild that combines upstream sources with a meta
+data and build instructions in a "spec file" and additional files and
+patches. It allows to bundle all this up into a source package and
+build the software into a binary package. The second part - the rpm
+utility - can then install, update or remove the packages after
+checking for consistency.
+
+RPM differs from special purpose package managers - like those
+targeting a specific programming language - in trying to not make
+assumptions about how software looks like or gets packaged. Packaging
+software can be messy and RPM accommodates for that.
+
+It still offers help to create a POSIX-like operation system. Macros
+parameterize locations to make the build instructions more generic and
+more easily transferable between distributions. Nevertheless RPM
+packages are targeting a specific distribution (and release thereof). They are
+typically not suited to be installed or even built elsewhere without
+tweaking. RPM acknowledges that different distributions have
+different goals, resulting in different design decisions. The
+specifics of the packages often reflect these.
+
+RPM as a upstream project still tries to keep distributions from
+diverging unnecessarily but is aware that these differences are the
+reason distributions exist in the first place.
+
+# RPM in the Software handling stack
```

Comment:
Maybe this could be a subheading? Looks strange among the toplevel ones, and also uses mixed caps :sweat_smile:

Reply:
Or not. I guess I'll just turn it into a subsection and leave it alone :smile: